src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,195 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+        size = 0;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            head = new Node<>(null, value, null);
+            tail = head;
+            size++;
+            return true;
+        }
+        Node<T> localNode = new Node<>(tail, value, null);
+        tail.next = localNode;
+        tail = localNode;
+        size++;
+        return true;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index < 0 || index > size) {
```

Comment:
`check(index)`

Reply:
I can't do it because this is another exception
![image](https://user-images.githubusercontent.com/52455671/101925527-9db92d00-3bda-11eb-83f5-3034a191fe44.png)

Reply:
done